docs/quick-start.md

```diff
@@ -74,23 +74,84 @@ CMS with the demo site.
 
 You should be able to view the site at [localhost:8070](http://localhost:8070)
 
-### Connecting Richie to OpenEdx
+### Basic - Connecting Richie to OpenEdx
 
 This project is pre-configured to connect with an OpenEdx instance started with
 [OpenEdx Docker](https://github.com/openfun/openedx-docker], which provides a ready to use
 docker-compose stack of OpenEdx in several flavors. Head over to
 [OpenEdx Docker README](https://github.com/openfun/openedx-docker#readme) for instructions on how
 to bootstrap an instance.
 
-To handle HTTP redirections between the 2 Docker Compose stacks during the authentication process,
-you should edit your local hosts definitions (_e.g._ `/etc/hosts` on a \*NIX system) to add `edx`
-as a `localhost` alias:
+#### 1. Run OpenEDX and Richie on the same domain
+
+Richie and OpenEDX must be on the same domain to work properly.
+To do that you have to edit your hosts file (_.e.g_ `/etc/hosts` on a \*NIX system) to alias
+a domain `local.dev` with two subdomains `richie` and `edx` to localhost:
 
 ```
 # /etc/hosts
-127.0.0.1 localhost edx
+127.0.0.1 richie.local.dev
+127.0.0.1 edx.local.dev
 ```
```

Comment:
Have you tried to use add a `dnsmasq` service in the docker-compose definitions to avoid having to modify system files, increase cross compatibility and automate this configuration step. Check out this project: https://github.com/jpillora/docker-dnsmasq

Reply:
Let's take a look at this 🤔

Reply:
You have to tweak our OS preferences to resolve dns queries with your dnsmasq container.

Reply:
Sic!